source/retryable-writes/retryable-writes.rst

```diff
@@ -168,19 +176,15 @@ transaction number for each supported write command in the batch.
 Retrying Write Commands
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-Drivers MUST NOT attempt to retry any write command that returns a response.
+Drivers MUST NOT attempt to retry any write command that does not encounter a
+retryable error.
```

Comment:
I realize this is a double negative. If anyone has a better way to word this, I'm open to suggestions.

Reply:
How about: "Drivers MUST only attempt to retry a write command that encounters a retryable error. Drivers MUST NOT attempt to retry a write command on any other error."